jsonchecker/__init__.py

```diff
@@ -35,15 +35,15 @@ def checker(self, seq, fname):
 
     def check_directory(self, directory):
         """Check one directory."""
-        if directory == '.':
-            directory = os.getcwd()
+        directory = os.path.abspath(directory)
         if os.path.isdir(directory):
+            last_directory = os.path.basename(os.path.normpath(directory))
+            if last_directory.startswith("."):
```

Comment:
the code uses single quotes, except for docstrings, so please change this to match that style